### ISSUE TITLE
Unable to train skills from skill trainer

### ISSUE THREAD

**Issue by rwendell** [ForgottenGlory/Living-Skyrim-3#620]
**LS Version**
3.6.0.1

**Describe the bug**
Skill Trainers are reading "enforced" level values from your "major skills" (classic classes and birthsigns) rather than actual skill levels.  This makes it impossible to train skills through the trainers.

**To Reproduce**
Steps to reproduce the behavior: Select a skill in ELSA or CCAB to "fortify" and then try to train that skill with a skill trainer.

**Expected behavior**
I expect to be able to train a skill from skill trainers

**Screenshots**
[screenshots.zip](https://github.com/ForgottenGlory/Living-Skyrim-3/files/7724631/screenshots.zip)

**Additional context**
@GravenImages was the one to point it out as a bug.  Message him for more context as I am unsure of how these mechanics clash together

[savefiles.zip](https://github.com/ForgottenGlory/Living-Skyrim-3/files/7724623/savefiles.zip)

**Comment by GravenImages** [comment]
Apologies, slight clarification; I believe that it's Classic Classes and Birthsigns causing this, not ELSA.

**Comment by Chaps-Fantastic** [comment]
[Save283_BB9A0BC3_0_4768756C2D566569207468652046726F67746F6E677565_RiverwoodAlvorsHouse_010051_20220117052459_14_1.zip](https://github.com/ForgottenGlory/Living-Skyrim-3/files/7879227/Save283_BB9A0BC3_0_4768756C2D566569207468652046726F67746F6E677565_RiverwoodAlvorsHouse_010051_20220117052459_14_1.zip)

Was asked in Discord to send this save file as I was having the same issue:
A few levels ago I went to train Destruction and when I tried It said that I needed to level before I can train. I had several levels to go before hitting the cap at that point in Destruction and I was at 16/30 so I had plenty of training I could do. 

I tried to go to other trainers (at the College and eventually at the Blue Palace) and they all said the same, regardless of the skill. I'm now at 16/60 and still can't buy training.
